client/components/tinymce/plugins/simple-payments/style.scss

```diff
@@ -105,3 +105,7 @@
 	flex: none;
 	margin-left: 16px;
 }
+
+.editor-simple-payments-modal__form .upload-image {
```

Comment:
nit: Could be a nested SCSS declaration inside `editor-simple-payments-modal__form`